client/my-sites/backup/status/simplified-i4.jsx

```diff
@@ -93,7 +94,13 @@ export const RealtimeStatus = ( { selectedDate } ) => {
 		lastBackupAttemptOnDate &&
 		mostRecentBackupEver.activityId === lastBackupAttemptOnDate.activityId;
 
-	const dailyDeltas = getDeltaActivities( [ lastBackupAttemptOnDate ].filter( ( i ) => i ) );
+	// If the latest backup for this date is a full backup,
+	// show details about what it contains
+	const dailyDeltas = getDeltaActivities(
+		isSuccessfulDailyBackup( lastBackupAttemptOnDate )
```

Comment:
This is throwing an error on my site because there is no backup yet for the current date (today).

Reply:
I mean, `lastBackupAttemptOnDate` is undefined.

Reply:
Fixed (I think) in 876b06a :-)